store/storetest/terms_of_service_store.go

```diff
@@ -43,10 +44,11 @@ func testGetLatestTermsOfService(t *testing.T, ss store.Store) {
 	u1.Username = model.NewId()
 	u1.Email = MakeEmail()
 	u1.Nickname = model.NewId()
-	store.Must(ss.User().Save(&u1))
+	_, err := ss.User().Save(&u1)
+	require.Nil(t, err)
 
 	termsOfService := &model.TermsOfService{Text: "terms of service", UserId: u1.Id}
-	_, err := ss.TermsOfService().Save(termsOfService)
+	_, err = ss.TermsOfService().Save(termsOfService)
```

Comment:
`require.Nil(t, err)` right after this.

Seems to be a bug in our original test as well.